integration/util.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/xichen2020/eventdb/parser/json/value"
 	"github.com/xichen2020/eventdb/query"
 	xtime "github.com/xichen2020/eventdb/x/time"
+	"github.com/xichen2020/eventdb/x/unsafe"
```

Comment:
should use safe?

Reply:
This is fine as it's usage is as follows and is self contained.

```
func parseTimestamp(v *value.Value, timestampFormat string) (int64, error) {
       str, err := v.String()
       b, err := v.Bytes()
       if err != nil {
               return 0, err
       }
       t, err := time.Parse(timestampFormat, str)
       t, err := time.Parse(timestampFormat, unsafe.ToString(b))
       if err != nil {
               return 0, err
       }
       return t.UnixNano(), nil
}
```

Reply:
Agreed that the usage is self contained but let's still use `safe` here. We should restrict usage of `unsafe` to a bare minimum (guideline being we should not use it unless the usage is simple, self-contained, and absolutely critical to performance).